init

```diff
@@ -34,6 +36,10 @@ if [[ $? = 4 ]]; then
   etcdctl set /environment/SCRIPTS-REV  $(cd $SCRIPTDIR && git branch|grep '\*'|cut -d' ' -f2 && cd -)
 fi
 
+if [[ "$VERSION" = "v3" ]]; then
```

Comment:
It mainly has to happen after etcd2-setup.sh runs, which for the control tier happens on line 25. I stuck it after environment because by then, we're 100% sure etcd is running/configured.